components/MapSection/utils/index.ts

```diff
@@ -41,7 +41,9 @@ export const getDataSuffix = ({ indicator, year }: SpotlightOptions): string | u
 export const setQuery = (router: NextRouter, options: SpotlightOptions, location?: SpotlightLocation): void => {
   const { route, push } = router;
   const { pathname } = window.location;
-  let as = `${pathname}?${THEME_QUERY}=${options.theme?.slug}&${INDICATOR_QUERY}=${options.indicator?.ddw_id}&${YEAR_QUERY}=${options.year}`;
+  let as = `${pathname}?${THEME_QUERY}=${options.theme ? options.theme.slug : ''}&${INDICATOR_QUERY}=${
+    options.indicator ? options.indicator.ddw_id : ''
+  }&${YEAR_QUERY}=${options.year ? options.year : ''}`;
```

Comment:
There should always be a theme, unless u've changed something to change that... hmm... as I keep telling u, I don't use those `?` lightly :)

Reply:
Removed unnecessary checks for undefined theme variable